XCUITests/PrivateBrowsingTest.swift

```diff
@@ -48,34 +51,35 @@ class PrivateBrowsingTest: BaseTestCase {
         navigator.openNewURL(urlString: url1)
         waitUntilPageLoad()
         navigator.goto(TabTray)
-        navigator.goto(BrowserTab)
-        navigator.goto(TabTray)
 
         waitforExistence(app.collectionViews.cells[url1Label])
         let numTabs = app.collectionViews.cells.count
```

Comment:
Consider putting this in the `userState`.

```swift
class FxUserState: UserState {
   …
   let numTabs: Int = 0
   …
}
```

```swift

    map.addScreenState(TabTray) { screenState in
        …
        screenState.onEnter { userState in 
            userState.numTabs = app.collectionViews.cells.count
        }
        …
    }
```

Reply:
👍 Added numTabs to UserState